apps/codeforafrica/src/lib/data/blockify/getInTouch.js

```diff
@@ -0,0 +1,15 @@
+import { actionFromActionButton } from "@/codeforafrica/lib/data/utils";
```

Comment:
@kelvinkipruto do we really need this file? I think we only need to rename GetInTouch block field as action.